.github/workflows/cleanup-webapp-test.yaml

```diff
@@ -46,3 +57,15 @@ jobs:
       - name: Deletion not Skipped
         run: "[[ \"${{steps.cleanup.outputs.skipped}}\" == \"false\" ]]"
 
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@main
+      with:
+        node-version-file: .nvmrc
+    - run: npm ci
```

Comment:
With npm, you can add the `--production` flag to skip installation of dev dependencies used for linting, testing, and development. Handy for creating build artifacts in less time which are more lightweight

Reply:
I had to revert this, since the build uses requires dev dependencies installed.

<img width="813" alt="image" src="https://user-images.githubusercontent.com/423234/214334817-8f618555-5eb7-4620-915b-091b6b62baf0.png">

Reply:
I don't think build dependencies belong into the production dependencies.

Reply:
Yep, that checks. Fair enough!